app/main/views/audits.py

```diff
@@ -180,6 +212,48 @@ def acknowledge_audit(audit_id):
     return jsonify(auditEvents=audit_event.serialize()), 200
 
 
+@main.route('/audit-events/<int:audit_id>/acknowledge-including-previous', methods=['POST'])
```

Comment:
> As for ease of log finding, I don't really see 123123...321321 making things much easier. 

That's not what I'm suggesting at all. I'm suggesting `POST /services/<id>/updates/acknowledge`.

I think it's much more likely that we'd need to change the service edits flow than us building an event acknowledgement flow for different types of events. And `/services/<id>/updates/acknowledge` feels like an easier endpoint to evolve for this as well as not forcing us to define operations we don't want to define at the moment (like acknowledging events of different types).